content/docs/postgresql/postgres-upgrade.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Upgrading your Postgres version
+subtitle: Learn how upgrade to a new major Postgres version in Neon
+enableTableOfContents: true
+updatedOn: '2024-08-07T21:36:52.673Z'
+---
+
+This topic describes how to upgrade your Neon project from one **major** Postgres version to a newer one.
+
+Postgres version numbers consist of a **major** and a **minor** version number. For example, in the version number 16.1, 16 is the major version number and the 1 is the minor version number.
+
+Neon manages **minor** Postgres version upgrades for you, as per the [Neon Postgres Version Support Policy](/docs/postgresql/postgres-version-policy). Typically, no user action is required for **minor** version upgrades. Neon deploys minor versions soon after they become available. However, upgrading to a new major Postgres version is a manual task that must be performed by you.
+
+Each Neon project is tied to a specific Postgres major version, which you select when creating a Neon project.
+
+You can check your current Neon project's Postgres version in the **Project settings** widget on **Project Dashboard** or by running the following query from the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor) or any SQL client connection to your database:
+
+```sql
+SELECT version();
+```
+
+## Before you begin
+
+- Review the [PostgreSQL Release Notes](https://www.postgresql.org/docs/current/release.html) for the new Postgres version. New major Postgres versions typically introduce some user-visible incompatibilities. Review the release notes for these changes. Although you can upgrade from one major Postgres version to another without upgrading to intervening versions, be sure to review the release notes for intervening versions should you decide to skip a version.
```

Comment:
```suggestion
- Review the [PostgreSQL Release Notes](https://www.postgresql.org/docs/current/release.html) for the new Postgres version. Major Postgres versions often introduce user-visible incompatibilities, so review the release notes for these changes. While you can upgrade directly to a new major Postgres version without going through each intermediate version, make sure you review the release notes for any skipped versions, as they may contain changes relevant to your upgrade.
```

Reply:
Is skipping versions an option? Or is it applied automatically when a compute restarts?